src/config/networks/__tests__/networks.test.ts

```diff
@@ -92,7 +92,10 @@ describe('Networks config files test', () => {
       const { id } = networkConfig.network
 
       // Then
-      const isValid = ETHEREUM_NETWORK[id]
+      const networkName = Object.keys(ETHEREUM_NETWORK).find((network) => {
+        return ETHEREUM_NETWORK[network] == id
+      })
+      const isValid = ETHEREUM_NETWORK[networkName || '']
```

Comment:
```suggestion
      const isValid = !!networkName
```

Reply:
If we use !!, if the networkName is not a valid network, isValid would return true.

Reply:
I might be reading it wrong but it seems you're first finding the key by comparing the value to the passed chainId, and then again looking up this key in the chains list (which is redundant). `networkName` is supposed to be either an existing key (valid), or a null if it couldn't be found (not valid).

Reply:
You're right, i wasn't looking at the initialization of networkName